src/main/java/Group15/WorkoutView.java

```diff
@@ -124,6 +124,12 @@ private static Pane createButtonPane(){
                     case "Edit Workout" -> {
                     }
                     case "Save" -> {
+                        try {
```

Comment:
Could me handle the exception in the method like Elias pointed out? That way our view is much cleaner